loginas/tests/tests.py

```diff
@@ -1,37 +1,18 @@
 # -*- coding: utf-8 -*-
-from __future__ import absolute_import, print_function, unicode_literals
-
-import unittest
 from datetime import timedelta
+from importlib import reload
+from urllib.parse import urlsplit
 
-import django
 from django.conf import settings as django_settings
 from django.contrib.auth.models import User
 from django.contrib.messages.storage.cookie import CookieStorage
 from django.core.exceptions import ImproperlyConfigured, PermissionDenied
 from django.test import Client, TestCase
 from django.test.utils import override_settings as override_settings_orig
+from django.urls import reverse
 from django.utils import timezone
 from loginas import settings as la_settings
 
-try:
```

Comment:
Hmm, does this code hurt anything? I'd rather not remove code that doesn't cost anything.

Reply:
No, but this is just in tests, so if nothing in the test matrix is failing it seems a bit pointless to keep.

This is all quite ancient stuff. `django.urls` was added in 2.0, `urlpase` hasn't been there since Python 2.7, and `importlib` was added back in Python 3.1.